sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -484,6 +484,8 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
         """.stripMargin)
       .load(testFile("test-data/postgresql/tenk.data"))
       .createOrReplaceTempView("tenk1")
+
+      session.sql("select * from tenk1 where stringu1 like stringu2 escape '\"'")
```

Comment:
This is not correct and I meant we need test s in `resources/sql-tests/inputs/`. Or, to check if the parser works well for the new token `ESCAPE`, its ok to add tests in `ExpressionParserSuite`.

Reply:
OK. I will remove this and add some tests in `ExpressionParserSuite`.